doc/neps/nep-0016-array-function-protocol.rst

```diff
@@ -0,0 +1,517 @@
+==================================================
+NEP: Dispatch Mechanism for NumPy's high level API
+==================================================
+
+:Author: Authors: Stephan Hoyer <shoyer@google.com> and Matthew Rocklin <mrocklin@gmail.com>
```

Comment:
I think the standard is one line/`:Author:` entry per author

Reply:
Done

Reply:
I pushed a few commits to your branch already, you probably want to pull them before making further edits :)